assets/js/translator.js

```diff
@@ -65,13 +65,18 @@ if(modeEnabled('translation')) {
         }
 
         function setupWebpageTranslation() {
+            if(window.location.href.indexOf('&qP=') != -1) {
```

Comment:
> In fact, right now translation gets triggered automatically on page load if there's text present, ideally we could easily modify that code to make it also do this.

It gets fired even if there's no text present. I believe I have figured out what causes the webpage translation to not fire up on page refresh. There is `translate()` call in `handlePairs()` method in `translator.js` file which gets fired on page refresh. Even when on webpage translation interface, on page refresh, this `translate()` gets called even before `showTranslateWebpageInterface()` completes and thus, on every page refresh, the if condition satisfies for translateText and `translateText()` gets called.